languageFactories/wdl-draft2/src/main/scala/languages/wdl/draft2/WdlDraft2LanguageFactory.scala

```diff
@@ -77,14 +83,16 @@ class WdlDraft2LanguageFactory() extends LanguageFactory {
       collectImportEvents
     }
 
-    val errorOr: ErrorOr[ValidatedWomNamespace] = (for {
+    val checked: Checked[ValidatedWomNamespace] = for {
       wdlNamespace <- wdlNamespaceValidation.toEither
       _ <- validateWorkflowNameLengths(wdlNamespace)
       importedUris = evaluateImports(wdlNamespace)
-      womExecutable <- wdlNamespace.toWomExecutable(Option(source.inputsJson))
+      womExecutable <- wdlNamespace.toWomExecutable(Option(source.inputsJson)) // wdlNamespace.toWomExecutable(Option(source.inputsJson))
       validatedWomNamespaceBeforeMetadata <- LanguageFactoryUtil.validateWomNamespace(womExecutable)
       _ <- checkTypes(wdlNamespace, validatedWomNamespaceBeforeMetadata.womValueInputs)
-    } yield validatedWomNamespaceBeforeMetadata.copy(importedFileContent = importedUris)).toValidated
+    } yield validatedWomNamespaceBeforeMetadata.copy(importedFileContent = importedUris)
+
+    val errorOr: ErrorOr[ValidatedWomNamespace] = checked.toValidated
```

Comment:
ew

Reply:
And then straight back `.toCheck`... yeah I think I can improve this